packages/@ourworldindata/components/src/styles/typography.scss

```diff
@@ -409,6 +409,18 @@ body {
     @include body-3-medium-italic-underlined;
 }
 
+@mixin body-3-bold {
+    font-family: $sans-serif-font-stack;
+    font-size: 0.875rem;
+    line-height: 1.5;
```

Comment:
Is this intentionally higher than the previous weights? I assume it might look janky in combination with other weights.

Reply:
Yeah, on Figma this is the weight for the bold variant. It's only used in one place for now so I don't think we have to worry about intermingling.

But I'll check with Marwa what she thinks, just in case 🙂